processing/src/main/java/io/druid/query/aggregation/AggregateCombiner.java

```diff
@@ -38,14 +38,14 @@
  * @see DoubleAggregateCombiner
  * @see ObjectAggregateCombiner
  */
-public interface AggregateCombiner extends ColumnValueSelector
+public interface AggregateCombiner<T> extends ColumnValueSelector<T>
```

Comment:
Will this cause extensions to need to be modified? or will extensions that include `AggregateCombiner` implementations still work the same?

Reply:
AggregateCombiner is added just recently